docs/src/main/paradox/docs/delta/api/current/assets/views/blazegraph/sparql/fetched.json

```diff
@@ -0,0 +1,33 @@
+{
+  "@context": [
+    "https://bluebrain.github.io/nexus/contexts/sparql.json",
+    "https://bluebrain.github.io/nexus/contexts/metadata.json"
+  ],
+  "@id": "https://bluebrain.github.io/nexus/vocabulary/myview",
+  "@type" : [
+    "SparqlView",
+    "View"
+  ],
+  "_constrainedBy": "https://bluebrain.github.io/nexus/schemas/views.json",
+  "_createdAt": "2021-04-18T17:10:22.748Z",
+  "_createdBy": "http://localhost/v1/realms/myrealm/users/Bob",
+  "_deprecated": false,
+  "_self": "http://localhost/v1/views/org/proj/nxv:myview",
```

Comment:
the links do not seem to match the environment, I was expecting `http://localhost:8080/v1`

Reply:
this seems to be the case in a lot of examples...